cmd/aries-js-worker/main.go

```diff
@@ -38,13 +43,30 @@ type result struct {
 
 // All supported command handlers.
 func handlers() map[string]map[string]func(*command) *result {
-	return map[string]map[string]func(*command) *result{
-		"test": {
```

Comment:
@sudeshrshetty can you leave these in there for now? So we have an easy way to test JS<->WASM integration?

Reply:
done

Reply:
done